Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -1267,9 +1276,9 @@ String ResourceLoadStatisticsStore::ensureAndMakeDomainList(const HashSet<Regist
 
 void ResourceLoadStatisticsStore::insertDomainRelationshipList(const String& statement, const HashSet<RegistrableDomain>& domainList, unsigned domainID)
 {
-    auto insertRelationshipStatement = m_database.prepareStatementSlow(makeString(statement, ensureAndMakeDomainList(domainList), " );"_s));
-
-    if (!insertRelationshipStatement || insertRelationshipStatement->bindInt(1, domainID) != SQLITE_OK) {
+    auto preparedStatement = m_database.prepareStatementSlow(makeString(statement, ensureAndMakeDomainList(domainList), " );"_s));
+    CheckedPtr insertRelationshipStatement = &(preparedStatement.value());
```

Comment:
This doesn't seem right. We need to check !preparedStatement before we access the value. Otherwise, we'd hit a release assert in `value()`.